src/components/Alert/Alert.module.css

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2023  New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/* TODO: Review entire file for semantic token definiton */
+
+.alert {
+  display: flex;
+  align-items: center;
+  gap: var(--cpd-space-3x);
+  padding: var(--cpd-space-4x);
+  border-radius: 12px;
+  border: 1px solid;
+}
+
+.alert[data-type="success"] {
+  color: var(--cpd-color-green-600);
+  background-color: var(--cpd-color-green-200);
```

Comment:
Are these colours back to front? light text on dark instead of dark text on light bg?

Reply:
They are back to front yes, but we should never use these colour scales directly.
Jano is defining semantic tokens, and we should consume those... but they were not defined at the time of creating this PR.